src/test/java/com/meta/cp4m/configuration/RootConfigurationTest.java

```diff
@@ -229,6 +259,26 @@ void validWA(@TempDir Path dir) throws IOException {
     config.toServicesRunner();
   }
 
+  @Test
+  void validWA_null_store(@TempDir Path dir) throws IOException {
```

Comment:
let's stick the convention of using CamelCase for methods and variables in Java https://www.javatpoint.com/camel-case-in-java